Ktisis/Overlay/SkeletonEditor.cs

```diff
@@ -223,7 +223,12 @@ public sealed class SkeletonEditor {
 							var parentPos = model->Position + parent.Rotate(model->Rotation) * model->Height;
 
 							Dalamud.GameGui.WorldToScreen(parentPos, out var pPos);
-							draw.AddLine(pos, pPos, boneColor, Ktisis.Configuration.SkeletonLineThickness);
+							uint lineColor = boneColor;
+							if (bone.Category != parent.Category) lineColor = ImGui.GetColorU32(Ktisis.Configuration.GetCategoryColor(parent));
```

Comment:
This is where the category overlap is being checked, but the result isn't as I would have expected.
![image](https://user-images.githubusercontent.com/107635249/190895813-1d21a1c6-c8b8-4b6d-a895-af5e4c915618.png)